js/mail.js

```diff
@@ -675,6 +683,16 @@ var Mail = {
 					});
 					// Remove spinner when loading finished
 					$('iframe').parent().removeClass('icon-loading');
+					// Add body content to inline reply (html mails)
+					var text = $(this).contents().find('body')[0].innerText;
+					if(!draft) {
+						document.getElementsByClassName('reply-message-body')[0].innerText = (
```

Comment:
This can be replaced by jQuery like ``$('.reply-message-body').first().text(...);`` I guess